Makefile

```diff
@@ -4,7 +4,7 @@
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
 IMAGE_OWNER ?= opendatahub
-VERSION ?= 2.1.0
+VERSION ?= 2.4.0
 # IMAGE_TAG_BASE defines the opendatahub.io namespace and part of the image name for remote images.
```

Comment:
I wonder for `incubation` branch we should have this set to a generic version so that we don't need to update this with every release.

Reply:
i was thinking to keep it, but then since we have "stable" may give the wrong feeling if anyone checkout the commit and see 2.1.0 with "stable".
probably after this 2.4.0 we can stay there for a while?

Reply:
Okay, that works as well. We can next update it for 3.x release